.github/issue_template/bug_report.md

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
I learned this the hard way — the directory needs to be named `ISSUE_TEMPLATE`. If you change the name but git doesn't pick it up, run:
```
$ git config core.ignorecase false
```
This is my bad for getting the first one wrong and not realizing

Reply:
Got it, will do!